set.mm

```diff
@@ -530574,10 +530578,10 @@ automatically a set (see ~ bj-mooreset ).  (Contributed by BJ,
 
 $)
 
-  $( Symbol for the nonnegative reals. $)
+  $( Token for the nonnegative reals. $)
   $c RR>=0 $.
 
-  $( Symbol for the (strictly) positive reals. $)
+  $( Token for the (strictly) positive reals. $)
   $c RR>0 $.
 
```

Comment:
What's wrong with "symbol"? According to the Metamath book (section 4.2.1) "... symbols are [user-defined] tokens ...". Actually, $c defines "(math) symbols" (see section 4.2.3), and "tokens" are more general (" ... tokens, which are strings of non-whitespace printable characters ... separated by white space ...", see section 4.2).

Reply:
No strong opinion.  To me, the symbol is the thing that is displayed on the webpage.  In other words, the typesetting definitions at the end of set.mm (by the way, the title of the appendix uses "token"), is of the form:
>htmldef "token" as "unicode-symbol";
  althtmldef "token" as "gif-symbol";
  latexdef "cadd" as "latex-symbol";

But again, not a golden rule to me.